core/src/main/java/bisq/core/trade/statistics/TradeStatistics3.java

```diff
@@ -437,7 +437,12 @@ public boolean isValid() {
         }
         long maxTradeLimit = Coin.COIN.multiply(2).value;
         try {
-            maxTradeLimit = PaymentMethod.getPaymentMethodById(getPaymentMethod()).getMaxTradeLimitAsCoin(currency).value;
+            // We cover only active payment methods. Retired ones will not be found by getActivePaymentMethodById.
+            String paymentMethod = getPaymentMethod();
+            Optional<PaymentMethod> optionalPaymentMethodById = PaymentMethod.getActivePaymentMethodById(paymentMethod);
+            if (optionalPaymentMethodById.isPresent()) {
+                maxTradeLimit = optionalPaymentMethodById.get().getMaxTradeLimitAsCoin(currency).value;
+            }
         } catch (Exception ignore) {
         }
```

Comment:
Maybe `log.warn(ignored)` exception,  just in case it hides a bug?

Reply:
Was thinking as well but the only area where it could throw is when the payment method is not in our defined list inside tradestats, which is an expected case when a user has not updated and new payment methods got added.

Reply:
``` public String getPaymentMethod() {
        try {
            return PaymentMethodMapper.values()[Integer.parseInt(paymentMethod)].name();
        } catch (Throwable ignore) {
            return paymentMethod;
        }
    }```

Reply:
Added a warn....